### DISCUSSION TITLE
Confirmation email sent to the "former" address

### DISCUSSION THREAD

**Discussion by 66leo** [supabase/supabase#21710] [Questions]
I am having difficulties with the confirmation email functionality. While the login, sign-up, and password reset processes are functioning as intended, the Email Update Confirmation sends to the old email address instead of the new one. Although the message correctly states the change from "old-address@example.com" to "new_address@example.com," the email is dispatched to the former address.
I use FlutterFlow the issue remains with both the built-in Auth action for Email Update and a brief segment of code.

`Future changeEmail(String newEmail) async {
  final response =
      await SupaFlow.client.auth.updateUser(UserAttributes(email: newEmail));

  if (response.user != null) {
    return true;
  } else {
    return false;
  }
}`

 I can reproduce from scratch, but confirmed with FF it's not a bug. I've contacted support and they suggested to ask here. Any idea?

![Screenshot 2024-03-01 at 13 31 09](https://github.com/supabase/supabase/assets/161693876/61ad0790-664d-4d4b-8abb-ac27cadb998b)

![Screenshot 2024-03-01 at 13 31 20](https://github.com/supabase/supabase/assets/161693876/67508ee7-c70a-4a38-acf5-e042e9e9a44f)

**Comment by GaryAustin1** [comment]
If you have this checked:
![image](https://github.com/supabase/supabase/assets/54564956/de4dfcbc-45a4-438a-8bb1-30967452600f)
Then it should send email to both.  Is the email for new somehow going to spam?

I'm not aware of any control you have over this in the client code.

**Reply by GaryAustin1** [reply]
You might look in the auth logs of the dashboard and see if there is an error sending the email, but I'm pretty sure it would abort back to you with an error to the client.

**Reply by 66leo** [reply]
I pressed some buttons and something moved but it's not fixed. I'll use the logs to navigate, thanks (super-beginner here).

**Comment by 66leo** [comment]
Thank you @GaryAustin1  for your reply. I do have it checked, and the second email is not in spam.
I know, it's weird. I was hoping for a miracle but worst-case scenario I'll go back to Supabase support.
